pkg/serverless/tags/testGetRuntimeRetries/13/environ

```diff
@@ -0,0 +1 @@
+AWS_EXECUTION_ENV=nodejs14.x
```

Comment:
Is this related to the PR?

Reply:
No. looks like this file gets generated when we run tests.

Reply:
Will remove